### PR TITLE
Fix: Fix docker tags

### PR DESCRIPTION
**Pull request by nepalevov**
This PR:
1. introduces the `is_latest` variable, which indicates if a version is semantically latest compared to the latest git tag; useful for docker tagging.
2. fixes docker tags logic: latest will be added only for the latest release build; add "development" tag for images built from "development" branch

### CODE REVIEW

File: actions/semantic_versioning/action.yml [hunk 1]
```diff
@@ -24,11 +27,15 @@ runs:
       id: semantic
       shell: bash
       run: |
+        #!/bin/bash
```
Comment (justrp) [inline actions/semantic_versioning/action.yml:30]: do we need that if we have `shell: bash` two lines above?

**Comment by justrp** [comment]
to me that looks good if you tested it out and it works :)

**Review by alexey-ban — APPROVED** [review]
Looks good

File: actions/semantic_versioning/action.yml (hunk 1 above)
Comment (alexey-ban) [reply actions/semantic_versioning/action.yml:30]: I think yes. It help us test script locally as is

**Pull request merged**